tests/lib/rules/no-trailing-spaces.js

```diff
@@ -19,8 +19,42 @@ var eslintTester = new ESLintTester(eslint);
 eslintTester.addRuleTest("lib/rules/no-trailing-spaces", {
 
     valid: [
-        "var a = 5;",
-        "var a = 5,\n    b = 3;"
+        {
+            code: "var a = 5;",
+            options: [{}]
+        },
```

Comment:
What I mean is omit `options` altogether. There's a difference between passing an array with an empty object and not passing anything.